core/modules/field/modules/text/text.module

```diff
@@ -232,28 +257,41 @@ function text_field_formatter_settings_summary($field, $instance, $view_mode) {
   $display = $instance['display'][$view_mode];
   $settings = $display['settings'];
 
-  $summary = '';
+  $summary = array();
 
   if (strpos($display['type'], '_trimmed') !== FALSE) {
-    $summary = t('Trimmed limit: @trim_length characters', array('@trim_length' => $settings['trim_length']));
+    $summary[] = t('Trimmed limit: @trim_length characters', array('@trim_length' => $settings['trim_length']));
+    if ($settings['word_boundary']) {
+      $word_boundary = t('Trim on word boundary.');
+      if ($settings['ellipsis']) {
+        $word_boundary .= ' ' . t('Add ellipsis.');
+      }
+      $summary[] = $word_boundary;
+    }
   }
 
-  return $summary;
+  return implode('<br />', $summary);
```

Comment:
Might be better to use `theme_item_list()` here. More readable as a list of text manipulations that will be applied.

Reply:
Not sure about this... Is there an example of where we output a bulleted list in this scenario elsewhere? Also, it's only two lines...

As for the wording, I was copying what I've seen elsewhere (e.g. `Image style: Large (800x600)` (for an image field)) - '[Label]: [Setting]'

Reply:
Sorry I missed this comment @BWPanda 

So yes, there are plenty of examples where we are using bullet point list for things to make them more readable. We sometimes add a check, so to count items and only render a bullet point list if more than one.

Here's a few that I could find quickly:
- https://github.com/backdrop/backdrop/blob/1.x/core/update.php#L359
- https://github.com/backdrop/backdrop/blob/1.x/core/authorize.php#L155
- https://github.com/backdrop/backdrop/blob/1.x/core/includes/theme.maintenance.inc#L206

...there's heaps more.

Reply:
@klonos I meant, is there an example of where we use bullet points in `hook_field_formatter_settings_summary()`? I ask because this is a specific place where vertical space is often limited (or at least taking up too much vertical space in these tables wouldn't look that great)...

![image](https://user-images.githubusercontent.com/2385329/143764893-c9d03635-3dd3-4441-abeb-2f4058108ada.png)

vs.

![image](https://user-images.githubusercontent.com/2385329/143764937-f8adb0d7-6bcc-42d2-857d-c8fe180c3a32.png)

The latter looks a bit out-of-place to me... Perhaps if this were implemented across the board, then it might be better, so maybe a separate issue for that?